src/main/java/com/onarandombox/MultiverseCore/utils/SimpleWorldPurger.java

```diff
@@ -32,16 +33,8 @@
 
     private MultiverseCore plugin;
 
-    private Class<Entity> ambientClass = null;
-
     public SimpleWorldPurger(MultiverseCore plugin) {
         this.plugin = plugin;
-        try {
-            Class entityClass = Class.forName("org.bukkit.entity.Ambient");
-            if (Entity.class.isAssignableFrom(entityClass)) {
-                ambientClass = entityClass;
-            }
-        } catch (ClassNotFoundException ignore) { }
```

Comment:
I'm curious as to why this was ever needed. The Ambient class has been present since at least Bukkit 1.5!

Reply:
mv code was from like 2011, and bukkit 1.5 was like in 2013, ig they just patched it and no one care to change it after that.